### PR TITLE
PLF-6880 Allow to add Container Lifecycle listeners to PortalContaine…

### PR DESCRIPTION
**Pull request by boubaker**
…r as for RootContainer

### CODE REVIEW

File: exo.kernel.container/src/main/java/org/exoplatform/container/PortalContainer.java [hunk 1]
```diff
@@ -632,6 +634,10 @@ public boolean isStarted()
    @Override
    public void start()
    {
+      if(!initialized_) {
+        initialized_ = true;
+        super.start(true);
```
Comment (thomasdelhomenie) [inline exo.kernel.container/src/main/java/org/exoplatform/container/PortalContainer.java]: If initialized_ is false, the super.start method will be called twice. Cannot it be a problem ? Why is this change needed ?

Comment (boubaker) [inline exo.kernel.container/src/main/java/org/exoplatform/container/PortalContainer.java]: No, it's not a problem because the flag started_ will be switched to true.
But you are right, I modified the implementation to make sure that it will be called once.
Thank you.

Comment (boubaker) [inline exo.kernel.container/src/main/java/org/exoplatform/container/PortalContainer.java]: Done

**Comment by boubaker** [comment]
Merged
